powervs-ipi-scripts/setup-access-group.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+#
+
+abort() {
+  echo "ERROR: $1 -- aborting..."
+  exit 1
+}
+
+# expects $ROLES to have the list of roles for the target service
+addPolicy() {
+  TMPF=$(mktemp)
+  $IAM access-group-policy-create $ACCESSGRP -q --roles "$ROLES" $* > $TMPF 2>&1
+  if [ $? -ne 0 ]; then
+    test "$DEBUG" && cat $TMPF || grep 'Error response' $TMPF | grep -v 'identical attributes already exist'
+  fi
+  rm -f $TMPF
+}
+
+# parse options specified
+while :; do
+  case $1 in
+    -a)
+      test "$2" && { ACCESSGRP=$2; shift; } || abort "no Access Group name specified for $1"
+      ;;
+    -d)
+      DEBUG=true
+      ;;
+    -h)
+      echo "$0 [-a access-group-name] [-r resource-group-name] [-u ibmid-user] [-d] [-h]"
+      echo "  -a: specifies the access group to populate with necessary policies, or uses the default name"
+      echo "  -r: specifies the resource group to tie the policies to, or defaults to the Default group"
+      echo "  -u: optionally adds the IBMid user to the access group"
+      exit 0
+      ;;
+    -r)
+      test "$2" && { RESOURCEGRP=$2; shift; } || abort "no Resource Group name specified for $1"
```

Comment:
I tried to set a resource group name that happens to have spaces in it, which didn't work due to word splitting. I think it needs to be enclosed in quotes everywhere it's used to prevent word splitting.

e.g.:
```
$  ic resource group "IBM Think 2024"
Retrieving resource group IBM Think 2024 under account <redacted> as jwcarman@us.ibm.com...
OK

                          
Name:                     IBM Think 2024
Account ID:               <redacted>
ID:                       <redacted>
Default Resource Group:   false
State:                    ACTIVE
```

Reply:
Good catch. Will probably have to do similar to `$ROLES` in `addPolicy`, but thanks!

Reply:
PTAL at the latest commit which allows space chars in RG & AG.